src/main/java/com/example/digitalmenuapi/model/AdminMenuItems.java

```diff
@@ -0,0 +1,106 @@
+package com.example.digitalmenuapi.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * Admin Menu Model class.
+ */
+@Document(collection = "AdminMenuItems")
+public class AdminMenuItems {
```

Comment:
You could have either extended the MenuItems class and added the new field 'available' here or possibly used the MenuItems object within here 'private MenuItem menuItem' 
You could also change the name to AdminMenuItem as this is only for 1 item.